jmclient/jmclient/wallet.py

```diff
@@ -889,6 +916,16 @@ def yield_imported_paths(self, mixdepth):
         """
         return iter([])
 
+    def _is_native_utxo(self, path):
```

Comment:
Perhaps add a couple of sanity check tests that the implementation of this function in the subclasses returns what's expected for different keys?

Reply:
done in b28a7fa

Reply:
Thanks. In https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/957/commits/b28a7fa5ce34d00004aee4160e5d3e0c366fc9a2 . Covers the cases I was thinking of.